src/metabase/driver/generic_sql/query_processor.clj

```diff
@@ -97,8 +114,9 @@
     (formatted (expression-with-name expression-name)))
 
   Field
-  (formatted [{:keys [schema-name table-name special-type field-name]}]
-    (let [field (keyword (hx/qualify-and-escape-dots schema-name table-name field-name))]
+  (formatted [unresolved-field]
```

Comment:
What do you mean be calling this a "unresolved" field?

Reply:
It needs to be resolved to an alias. Makes sense to me, but I can change it if you like.

Reply:
It might make sense to think of a slightly different name. We use "unresolved Field" in an earlier stage in the query processor to refer to Fields that are referenced in the query that we haven't fetched from the DB yet. So this could end up getting a little confusing.